dpsim-villas/src/InterfaceWorkerVillas.cpp

```diff
@@ -215,15 +220,29 @@ void InterfaceWorkerVillas::readValuesFromEnv(std::vector<Interface::AttributePa
 						mCurrentSequenceInterfaceToDpsim++;
 					}
 				}
+
+				if (!pollFds.empty()) {
+					//Manually clear the event file descriptor since Villas does not do that for some reason
```

Comment:
Do we have an issue for that in villas?

Reply:
I do not think so, I will create one and link it in the code.

Reply:
Linked to issue in d2ca027e3d0446a8fc232bf4f5c2ad120f0c9f22

Reply:
https://git.rwth-aachen.de/acs/public/villas/node/-/issues/347